### PR TITLE
Improving CSV reports performance

### PR DESCRIPTION
**Pull request by superalex**

### CODE REVIEW

File: taiga/projects/userstories/services.py [hunk 1]
```diff
@@ -143,9 +144,28 @@ def userstories_to_csv(project,queryset):
                    "tags",
                    "watchers", "voters"]
 
-    for custom_attr in project.userstorycustomattributes.all():
+    custom_attrs = project.userstorycustomattributes.all()
+    for custom_attr in custom_attrs:
         fieldnames.append(custom_attr.name)
 
+    roles = project.roles.filter(computable=True).order_by('name')
```
Comment (jespino) [inline taiga/projects/userstories/services.py]: I know was wrong in the original version, but if you want to get the csv result ordered, you must order by slug, which is the field that is used to show the points in the result.

**Comment by jespino** [comment]
Looks great, I going to test it manually and if everythings works correctly and you fix my comment I will merge it :+1:

**Comment by jespino** [comment]
Ready! :+1:

**Comment by bameda** [comment]
And backported??

**Comment by jespino** [comment]
Nope

**Pull request merged**